webhook/psbinding/contexts.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2019 The Knative Authors
```

Comment:
<!--copyright-bot-->
Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

Reply:
oops... guess we're not in Kansas anymore...